website/docs/d/oracle_autonomous_database_backup.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Oracle"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_oracle_autonomous_database_backup"
+description: |-
+    Gets information about an existing Autonomous Database Backup.
+---
+
+# Data Source: azurerm_oracle_autonomous_database_backup
+
+Use this data source to access information about an existing Autonomous Database Backup.
+
+## Example Usage
+
+```hcl
+data "azurerm_oracle_autonomous_database_backup" "example" {
+  autonomous_database_id = azurerm_oracle_autonomous_database.example.id
+}
+
+```
+
+## Arguments Reference
+The following arguments are supported:
+
+* `autonomous_database_id` - The azureId of the Autonomous Database for we will be listing the backups.
+
+## Attributes Reference
+In addition to the Arguments listed above—the following Attributes are exported:
+
+* `id` - The ID of the Autonomous Database Backup.
```

Comment:
Could we update the documentation to reflect the data source In its current "plural" form? Its top level attributes are the database ID and an `autonomous_database_backups` list, which then has the following properties.